python/graphscope/nx/tests/test_copy_on_write.py

```diff
@@ -262,6 +289,10 @@ def setup_method(self):
         #     ).values
         # )
 
+    def test_error_with_multigraph(self):
+        with pytest.raises(NetworkXError):
```

Comment:
Better to also specify the `matches` parameter, for instance, match the `parallel edge`